lib/slack.py

```diff
@@ -85,8 +89,7 @@ def suggest_video(self, query, channel, user, is_shuffle):
             params['replace_original'] = True
             return params
         else:
-            client = self.get_client()
-            response = client.api_call("chat.postEphemeral", **params)
+            response = self.send_message(params, True)
```

Comment:
Sometimes, for clarity, it's nice to do `self.send_message(params, is_ephemeral=True)` just so other developers don't need to track down the function definition to figure out what argument you're passing as `True`